pom.xml

```diff
@@ -99,6 +99,7 @@
         <testcontainers.version>1.17.5</testcontainers.version>
         <!-- stay 7.5 for JDK 8 support -->
         <testng.version>7.5</testng.version>
+        <apache.httpclient.version>4.5.13</apache.httpclient.version>
```

Comment:
The legacy driver was using http client 4.5 and it's known to have "no response" error sometimes. Can we [migrate to 5.2](https://hc.apache.org/httpcomponents-client-5.2.x/migration-guide/preparation.html)?